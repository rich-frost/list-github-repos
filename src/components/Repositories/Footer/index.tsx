import { Tr, Td, Tfoot, Text, Flex } from '@chakra-ui/react';

export default function Footer() {
  return (
    <Tfoot>
      <Tr>
        <Td colSpan={3}>
          <Flex justify="flex-end">
            <Text as="i">
              Showing top 10 GitHub repos for topic, sorted by best match
            </Text>
          </Flex>
        </Td>
      </Tr>
    </Tfoot>
  );
}
