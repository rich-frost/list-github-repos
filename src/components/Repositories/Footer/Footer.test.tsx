import renderer from 'react-test-renderer';
import Footer from './index';
import { Table } from '@chakra-ui/react';

describe('Footer component', () => {
  it('renders the component', () => {
    const component = renderer.create(
      <Table>
        <Footer />
      </Table>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
