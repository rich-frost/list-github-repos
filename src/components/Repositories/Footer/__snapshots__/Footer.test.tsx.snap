// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the component 1`] = `
<table
  className="chakra-table css-0"
>
  <tfoot
    className="css-0"
  >
    <tr
      className="css-0"
    >
      <td
        className="css-0"
        colSpan={3}
      >
        <div
          className="css-1bvc4cc"
        >
          <i
            className="chakra-text css-0"
          >
            Showing top 10 GitHub repos for topic, sorted by best match
          </i>
        </div>
      </td>
    </tr>
  </tfoot>
</table>
`;
